main.rb

```diff
@@ -1,10 +1,10 @@
-require './app'
+require './menu'
 
 def main
   puts "\nWelcome to School Library App!\n"
 
-  app = App.new
-  app.main
+  menu = Menu.new
+  menu.display_list_of_options
 end
```

Comment:
- [ ] I could not test the app functionalities requested by the previous reviewer, Because I'm getting the below error when starting the app. Please make sure to fix this issue and users should be able to start the app and preserve data functionalities should work as expected.

![image](https://github.com/aima98/OOP-School-Library1/assets/83803414/423fceb1-0e3a-4d31-8786-66de16e6ad22)